libwm/Wm4System.cpp

```diff
@@ -21,16 +21,9 @@ using namespace Wm4;
 #include <sys/stat.h>
 
 // support for GetTime
-#ifdef __APPLE__
 #include <sys/time.h>
 static timeval gs_kInitial;
 static bool gs_bInitializedTime = false;
-#else
-#include <sys/timeb.h>
-static long gs_lInitialSec = 0;
-static long gs_lInitialUSec = 0;
-static bool gs_bInitializedTime = false;
-#endif

```

Comment:
I'll admit I haven't looked at the intent of the code, but why is the Apple-specific branch now hardcoded?

Reply:
Pasting from [Actions](https://github.com/FluidityProject/fluidity/runs/7208479003?check_suite_focus=true#step:4:13547):
```
#15 5.116         CXX Wm4System.cpp
#15 5.709 Wm4System.cpp: In static member function 'static double Wm4::System::GetTime()':
#15 5.709 Wm4System.cpp:127:14: warning: 'int ftime(timeb*)' is deprecated: Use gettimeofday or clock_gettime instead [-Wdeprecated-declarations]
#15 5.709   127 |         ftime(&kTB);
#15 5.709       |         ~~~~~^~~~~~
#15 5.709 In file included from Wm4System.cpp:29:
#15 5.709 /usr/include/x86_64-linux-gnu/sys/timeb.h:29:12: note: declared here
#15 5.709    29 | extern int ftime (struct timeb *__timebuf)
#15 5.709       |            ^~~~~
```
I believe that is why I made the changes you highlighted.